<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Main Feed</string>
    </property>
    <widget class="QWidget" name="AppFeed" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>445</width>
       <height>445</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start video</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Alternate</string>
    </property>
    <widget class="QWidget" name="AppFeedAlt" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>445</width>
       <height>445</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="showMazeButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Maze</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputTextBox">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>254</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="InputGroupBox">
    <property name="geometry">
     <rect>
      <x>959</x>
      <y>543</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Command Input</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="imgAlgorithm_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>convert color</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="imgAlgorithm_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>126</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>find in range</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="imgAlgorithm_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>152</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>erosion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="imgAlgorithm_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>178</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>dialation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="textSubmitButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="HColorSliderLow">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>H color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSlider" name="SColorSliderLow">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>630</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>S Color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSlider" name="VColorSliderLow">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>670</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>V Color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>590</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>630</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>670</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(-5,-3)</string>
    </property>
   </widget>
   <widget class="QLabel" name="directionLabel">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>580</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="QLabel" name="distanceTravelledLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>585</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Location     Direction    Move Count     Range              RobotIP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>961</x>
      <y>391</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Command History</string>
    </property>
   </widget>
   <widget class="QSlider" name="VColorSliderHigh">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>670</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>V Color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSlider" name="HColorSliderHigh">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>590</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>H color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSlider" name="SColorSliderHigh">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>630</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>S Color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="lowRangeLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>710</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Range = (0,0,0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="highRangeLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>710</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>high Range = (180,360,360)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Robot</string>
    </property>
   </widget>
   <widget class="QComboBox" name="colorSelection">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="robotIPadrLabel">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>590</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="problemDescription">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>100</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>234</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>961</x>
      <y>79</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Problem Description</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showWaypointsButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Waypoints</string>
    </property>
   </widget>
   <widget class="QLabel" name="rangeLabel">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>585</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rangeSensorButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>620</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Range Sensor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showHSVslidersButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Color Ranges</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="networkGroupBox">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>100</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.41</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>10.10.1.16</string>
     </property>
    </widget>
    <widget class="QPushButton" name="radialSubmitButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.91</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>10.10.1.14</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.42</string>
     </property>
    </widget>
    <widget class="QLabel" name="networkLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select the robot's IP address</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scanNetworkButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan Network</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="commandHistory">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>410</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rangeLabel_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>590</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuRobotIP">
     <property name="title">
      <string>robotIP</string>
     </property>
     <addaction name="action0_0_0_0"/>
    </widget>
    <addaction name="menuRobotIP"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action0_0_0_0">
   <property name="text">
    <string>0.0.0.0</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
